android/README.md

```diff
@@ -2,9 +2,18 @@
 
 ## Prerequisites
 * Android Studio 3.3.2+
-* Java SE Development Kit 8
+* Java SE Development Kit 11 (Java 9 or newer)
```

Comment:
I was able to build this branch with the existing Java 8.  @mcdurdin , sounds like you prefer we stay with Java 8 for now? If so, this readme can revert.

I'll copy this to a future issue so we'll remember the steps to upgrade.

Reply:
Given the existing stable build doesn't build with Java 9+ it'd be better not to update if we don't have to.

Reply:
@safolit Since we're sticking with Java 8 for now, you can revert this README.

I've documented your steps for updating to Java 9+ in #2293

Reply:
I leave the steps about installation of openJDK there and updated it for openJDK 1.8.
Tested it locally with openJDK and it works well.